### DISCUSSION TITLE
Add UI configuration

### DISCUSSION THREAD

**Discussion by fabioc-aloha** [microsoft/sample-app-aoai-chatGPT#188] [Ideas]
Can you please add configurable parameters to replace the strings "Start chatting" and "This chatbot is configured to answer your questions" in the chat page?
That would make it super easy to personalize the chat ui!

Thanks,

Fabio

**Comment by rjhodder** [comment]
Agreed. When in demo mode it would be nice to notify users of expectations.

**Comment by rhematt** [comment]
This is in the environment variables under the UI sections.

The updated info includes:

> The interface allows for easy adaptation of the UI by modifying certain elements, such as the title and logo, through the use of [environment variables](https://github.com/microsoft/sample-app-aoai-chatGPT/discussions/188#environment-variables).
> 
> UI_TITLE
> UI_LOGO
> UI_CHAT_TITLE
> UI_CHAT_LOGO
> UI_CHAT_DESCRIPTION
> UI_FAVICON
> UI_SHOW_SHARE_BUTTON

**Reply by tomasdore** [reply]
Is there a way to use HTML in these variables, for instance so that I can add a link to a feedback form? I find that it doesn't show up correctly, or shows the HTML tags in the interface, if I add to UI_CHAT_DESCRIPTION.

**Reply by erolmam-75** [reply]
This is very good suggestion. Would be great with possibility to add something clickable. 
Now it is not even possible to add a "newline/Linebreak" in the text.
